apps/api/src/campaign-application/campaign-application.controller.ts

```diff
@@ -76,8 +77,36 @@ export class CampaignApplicationController {
   }
 
   @Get('byId/:id')
-  findOne(@Param('id') id: string) {
-    return this.campaignApplicationService.findOne(id)
+  async findOne(@Param('id') id: string, @AuthenticatedUser() user: KeycloakTokenParsed) {
+    const person = await this.personService.findOneByKeycloakId(user.sub)
+    if (!person) {
+      Logger.error('No person found in database')
+      throw new NotFoundException('No person found in database')
+    }
+    let isAdminFlag
+    if (!isAdmin(user)) {
+      isAdminFlag = false
+    } else {
+      isAdminFlag = true
+    }
```

Comment:
No need for the isAdminFlag. isAdmin already returns a boolean.
You can just do `const isAdmin = isAdmin(user)` and pass isAdmin to the the findOne service.